rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/TclLauncher.xtend

```diff
@@ -3,11 +3,10 @@ package org.testeditor.rcp4.tcltestrun
 import java.util.Map
 import org.eclipse.core.resources.IProject
 import org.eclipse.core.runtime.IProgressMonitor
-import org.eclipse.jface.viewers.IStructuredSelection
 
 interface TclLauncher {
 
-	public def LaunchResult launchTest(IStructuredSelection selection, IProject project, String elementId,
+	public def LaunchResult launchTest(String testCasesCommaList, IProject project, 
```

Comment:
I would prefer a proper `List<String>` for the names of the test-cases. If the list has to be passed to Maven in a comma separated way is for the concrete implementation to decide.